src/components/SlateEditor/plugins/blockPicker/SlateBlockPicker.tsx

```diff
@@ -67,7 +67,13 @@ const SlateBlockPicker = (props: Props & tType) => {
   const onInsertBlock = (block: Element) => {
     const { editor } = props;
     setIsOpen(false);
-    ReactEditor.focus(editor);
+    if (block.type === 'embed') {
+      setTimeout(() => {
+        ReactEditor.focus(editor);
+      }, 0);
+    } else {
```

Comment:
Eg veit ikkje om det er denne sin feil, men om du lagrer en artikkel med embed/h5p så scrolles det til embeden etter litt tid. Det ser litt rart ut når det ikkje er der du har jobba sist. Ellers synes eg dette ser bra ut.

Reply:
Slet litt med å framprovosere dette. Det er et lite øyeblikk etter artikkelen lagres at embeds lastes inn på nytt og dermed ikke fyller like mye plass, så det oppleves som at siden scroller. Utover dette skal ikke embeds ha noe påvirkning på scroll / focus.

Koden over er en 0ms forsinkelse ved innsetting av embed for å avvente fokus av editor til rett etter embeden har blitt plassert inn. H5P krasjet uten denne forsinkelsen.